distributedlog-protocol/src/main/java/org/apache/distributedlog/LogRecord.java

```diff
@@ -184,7 +192,23 @@ protected void setTransactionId(long txid) {
      * @return payload of this log record.
      */
     public byte[] getPayload() {
```

Comment:
why is this still necessary? back compat?

Reply:
I kept it there for back compat.